package.json

```diff
@@ -112,6 +114,7 @@
         "es6-promise-pool": "^2.5.0",
         "eslint": "^8.11.0",
         "eslint-plugin-mistica-local-rules": "workspace:*",
+        "eslint-plugin-storybook": "^0.6.15",
```

Comment:
what's this?

Reply:
It was installed automatically when upgrading storybook version. I don't think we need it, so maybe I can remove it